app/views/layouts/base.html.erb

```diff
@@ -52,11 +52,22 @@
     </div>
 
     <%= govuk_footer(
-      meta_items_title: 'Footer links',
-      meta_items: {
-        'Accessibility': '/accessibility',
-        'Cookies': '/cookies',
-        'Privacy': '/privacy',
-      }) %>
+        meta_items_title: 'Footer links', 
+        meta_items: {
+          'Accessibility': '/accessibility',
+          'Cookies': '/cookies',
+          'Privacy': '/privacy'
+        } ) do |footer| %>
+      <% footer.navigation do %>
+        <% if current_teacher %>
+          <div class="govuk-footer__section">
+            <h3>Get help</h3>
+            <p>If you're having difficulty completing your application form, contact: <a class="govuk-link"
```

Comment:
```suggestion
            <p>If you‘re having difficulty completing your application form, contact: <a class="govuk-link"
```

Also on line 67 in this file and 4 and 5 in the show view.

Reply:
Done 👍 
Removed the content_for from the show view as we aren't using 'yield' anymore so it is no longer necessary